doc/api/fs.md

```diff
@@ -3557,6 +3557,52 @@ It is possible to configure the console to render UTF-8 properly by changing the
 active codepage with the `chcp 65001` command. See the [chcp][] docs for more
 details.
 
+## fs.writev(fd, buffers, [position], callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fd` {integer}
+* `buffers` {ArrayBufferView[]}
+* `position` {integer}
+* `callback` {Function}
+  * `err` {Error}
+  * `bytesWritten` {integer}
+  * `buffers` {ArrayBufferView[]}
+
+Write an array of `ArrayBufferView` to the file specified by `fd` using `writev`.
+
+`position` refers to the offset from the beginning of the file where this data
+should be written. If `typeof position !== 'number'`, the data will be written
+at the current position.
+
+The callback will be given three arguments `(err, bytesWritten, buffers)` where
+`bytesWritten` specifies how many _bytes_ were written from `buffers`.
+
+If this method is invoked as its [`util.promisify()`][]ed version, it returns
+a `Promise` for an `Object` with `bytesWritten` and `buffers` properties.
+
+It is unsafe to use `fs.writev()` multiple times on the same file without
+waiting for the callback. For this scenario, [`fs.createWriteStream()`][] is
+recommended.
+
+On Linux, positional writes don't work when the file is opened in append mode.
+The kernel ignores the position argument and always appends the data to
+the end of the file.
+
+## fs.writevSync(fd, buffers, [position])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fd` {integer}
+* `buffers` {ArrayBufferView[]}
+* `position` {integer}
+* Returns: {number} The number of bytes written.
+
+For detailed information, see the documentation of the asynchronous version of
+this API: [`fs.writev(fd, buffer...)`][].
```

Comment:
Missing bottom reference.

Reply:
I do not understand.

Reply:
Currently, this link is not rendered as a link, you need to place this reference at the file bottom:
```md
[`fs.writev(fd, buffer...)`]: #fs_fs_writev_fd_buffer_position_callback
```

As references are sorted in ASCII order, it needs to be placed after this one:
```md
[`fs.writeFile()`]: #fs_fs_writefile_file_data_options_callback
```